fastmri_recon/evaluate/scripts/paper_eval_oasis.py

```diff
@@ -39,8 +39,8 @@ def evaluate_paper_oasis(n_samples=200):
         val_split=0.1,
     )
     val_gen_mask = train_gen.val_sequence
-    n_train = 1000
-    n_val = n_samples
+    n_train = min(1000, len(train_gen.filenames))
```

Comment:
This is a minute update here to ensure that we dont cross the number of files

Reply:
makes sense